client/client.go

```diff
@@ -515,6 +515,68 @@ func (r *NotaryRepository) GetChangelist() (changelist.Changelist, error) {
 	return cl, nil
 }
 
+// GetRepoKeyIDs returns a map of associations between this repo's roles and its key IDs
+func (r *NotaryRepository) GetRepoKeyIDs() (map[string][]string, error) {
+	// Update to latest repo state
+	_, err := r.Update(false)
```

Comment:
We can probably get away with skipping the update here, since `Update` only pulls down the tuf files and this is just looking at keys on disk.